src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -2782,6 +2814,12 @@ export default defineComponent({
       ignoreErrors = true
 
       if (ui) {
+        if (ui.getControls()) {
+          // save the state of player settings to reinitialize them upon next creation
+          const controls = ui.getControls()
+          emit('player-destroyed', controls.isFullScreenEnabled(), fullWindowEnabled.value, controls.isPiPEnabled())
+        }
```

Comment:
As this function is called by the Watch page, we could probably just return these three values in an object as the return value of the function and avoid the extra emit.